plugins/modules/irc.py

```diff
@@ -96,6 +102,16 @@
       - Text style for the message. Note italic does not work on some clients
     choices: [ "bold", "underline", "reverse", "italic", "none" ]
     default: none
+  validate_certs:
+    description:
+      - If set to V(false), the SSL certificates will not be validated.
+      - This should always be set to V(true). Using V(false) is unsafe and should only be done
+        if the network between between Ansible and the IRC server is known to be safe.
+      - B(Note:) for security reasons, you should always set O(use_tls=true) and
+        O(validate_certs=true) whenever possible.
+    default: false
```

Comment:
we recommend setting `true` because of security, so IMO default should be `true`

Reply:
i.e. force people to think about it

Reply:
Yes, but that would be a breaking change. The idea is to deprecate the current default not too far in the future, and then switch to the new (better) default eventually.

Reply:
I created #7578 for deprecating the current defaults.